platform/rct/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -66,27 +68,27 @@ static int initQueues()
     int success = 1;
 
     /* Create the queues used to hold Rx and Tx characters. */
-    xUsart0Rx = xQueueCreate(UART_QUEUE_LENGTH,
+    xUsart0Rx = xQueueCreate(UART_RX_QUEUE_LENGTH,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
-    xUsart0Tx = xQueueCreate(UART_QUEUE_LENGTH + 1,
+    xUsart0Tx = xQueueCreate(UART_TX_QUEUE_LENGTH + 1,
```

Comment:
Lets get rid of this annyoing `+ 1` crap here since it invalidates the point of using the `#define` above.